### ISSUE TITLE
Information related to training

### ISSUE THREAD

**Issue by PotatoSpudowski** [dchen236/FairFace#1]
Hi I hope you are doing well, 

I have been trying to work on something like this and I was wondering if you could provide more information related to your training performance. I have been facing difficulties getting a decent accuracy using the classifier.

Any information related to the number of epochs, batch size, learning rate and training loss would be appreciated.
